playbooks/setup_forklift.yml

```diff
@@ -37,5 +37,5 @@
 
     - name: 'Configure Forklift'
       copy:
-        content: "{{ forklift_config | to_nice_yaml }}"
+        content: "{{ forklift_config | to_json | from_json | to_nice_yaml }}"
```

Comment:
Why is this needed?

Reply:
`ansible-lint` (and only that, `ansible-playbook` is fine) fails to parse the playbook otherwise. I'll open a bug against `ansible-lint`, but so far this workaround was the cheapest to get it running.

Reply:
Mind adding that as a comment, ideally with a link to the ansible-lint issue?

Reply:
done